package.json

```diff
@@ -1,22 +1,54 @@
 {
   "name": "node-easypost",
   "description": "EasyPost Node Client Library",
-  "version": "2.1.1",
+  "version": "3.0.0",
   "author": "Sawyer Bateman <support@easypost.com>",
```

Comment:
Maybe I'll change it to "Easypost Engineering" or something, and point at the support email?